src/assets/i18n/fr.json5

```diff
@@ -772,18 +799,34 @@
   // "admin.metadata-import.breadcrumbs": "Import Metadata",
   "admin.metadata-import.breadcrumbs": "Importer Métadonnées",
 
+  //"admin.batch-import.breadcrumbs": "Import Batch",
+  "admin.batch-import.breadcrumbs": "Importation en lots",
+
   // "admin.metadata-import.title": "Import Metadata",
   "admin.metadata-import.title": "Importer Métadonnées",
 
   // "admin.metadata-import.page.header": "Import Metadata",
   "admin.metadata-import.page.header": "Importer Métadonnées",
 
+  //"admin.batch-import.page.header": "Import Batch",
+  "admin.batch-import.page.header": "Importation en lots",
+
   // "admin.metadata-import.page.help": "You can drop or browse CSV files that contain batch metadata operations on files here",
   "admin.metadata-import.page.help": "Vous pouvez glisser-déposer ou rechercher ici des fichiers CSV qui contiennent des opérations en lot sur les métadonnées",
 
+//"admin.batch-import.page.help": "Select the Collection to import into. Then, drop or browse to a Simple Archive Format (SAF) zip file that includes the Items to import",
+"admin.batch-import.page.help": "Sélectionner la collection dans laquelle importer les données. Ensuite, ajouter le fichier le fichier zip Simple Archive Format (SAF) contenant tous les Items à importer.",
```

Comment:
Should be : "Sélectionner la collection dans laquelle importer les données. Ensuite, ajouter le fichier zip Simple Archive Format (SAF) contenant tous les Items à importer."

Reply:
Done in last commit. 46df7e805e2eac71c31acd35bd15367ceb5efc46